app/javascript/packs/postcodesHelper.js

```diff
@@ -0,0 +1,45 @@
+// Help with finding constituencies from postcodes
+
+//display results on page
+const displayData = postcode => {
+  var onsId = postcode.result.codes.parliamentary_constituency;
```

Comment:
Consider `const` instead of `var` for these variables (same below), otherwise all looks good to me!